lib/utils/plugins.js

```diff
@@ -69,6 +69,7 @@ const clearModulesCache = () => {
 };
 
 const getPluginName = path => window.require('path').basename(path);
+const getPluginVersion = path => window.require(window.require('path').resolve(path, 'package.json')).version;
```

Comment:
This should probably be in a try/catch, just to be safe 😄  maybe if a plugin is not installed yet etc?